eng/pipelines/common/global-build-job.yml

```diff
@@ -119,6 +119,13 @@ jobs:
         inputs:
           artifact: Mono_Offsets_${{monoCrossAOTTargetOS}}
           path: '$(Build.SourcesDirectory)/artifacts/obj/mono/offsetfiles'
+    
+    - ${{ if eq(parameters.buildingOnSourceBuildImage, true) }}:
+      - template: /eng/common/templates/steps/source-build.yml
+        parameters:
+          platform:
+            buildScript: $(_sclEnableCommand) $(Build.SourcesDirectory)$(dir)build$(scriptExt)
```

Comment:
Is that the right command to invoke? cc @MichaelSimons

Reply:
I think so.  This is just the build script, the template defaults to specify all of the correct args.

Reply:
And is the _sclEnableCommand still needed? https://github.com/dotnet/runtime/blob/d841f5e17897be5ef6aef5031b8a03ed3d8fd8dd/eng/pipelines/common/global-build-job.yml#L100

Reply:
yes otherwise we get clang failures.